README.md

```diff
@@ -26,9 +26,9 @@ in your `/turing` directory on your laptop (or whatever directory you want this
 
 ```
 // using ssh keys
-git clone git@github.com:turingschool/ruby-exercises.git
+git clone git@github.com:josh-works/ruby-exercises.git
 // using https, if the above doesn't work:
-git clone https://github.com/turingschool/ruby-exercises.git
```

Comment:
@josh-works Did you mean to change this?

Reply:
ugh, no, this PR picked up an unwanted commit. I'll fix it. Thanks for highlighting!